code/socialDistribution/github_activity/github_activity.py

```diff
@@ -0,0 +1,32 @@
+import requests, timeago
+from datetime import timezone, datetime
+from .github_events import EventFactory
+
+REQUEST_URL = "https://api.github.com/users/{github_user}/events"
+
+event_factory = EventFactory()
+
+def get_event_description(event_name, event_data):
```

Comment:
should we call this set_event_description? pls, consider adding a short block comment describing functions/methods.